src/main/java/task_classes/Task.java

```diff
@@ -0,0 +1,28 @@
+package task_classes;
+
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+
+    public void setDone(){
```

Comment:
I think it would be nice to have a space between the parentheses for the arguments and the curly brace